src/test/shared/extensionUtilities.test.ts

```diff
@@ -23,4 +29,54 @@ describe('extensionUtilities', () => {
             assert.strictEqual(safeGet(undefined as Blah | undefined, x => x.someProp), undefined)
         })
     })
+
+    describe('createWelcomeWebview', async () => {
+
+        const context = new FakeExtensionContext()
+        let filepath: string | undefined
+
+        before(async () => {
+            const tempDir = await filesystemUtilities.makeTemporaryToolkitFolder()
+            context.extensionPath = tempDir
+        })
+
+        afterEach(async () => {
+            if (filepath && await filesystemUtilities.fileExists(filepath)) {
```

Comment:
Why not create `tempDir` and set extensionPath before _each_ test, then unconditionally delete tempDir after each test? Many tests do this on a per test basis. This will simplify the setup/teardown, and eliminate the risk of side effects. 

If you're concerned about extensionPath not referencing a valid location, it can be set to undefined after each test as well.